dist/web3.js

```diff
@@ -1444,6 +1444,9 @@ web3.setProvider = function (provider) {
     this.currentProvider = provider;
     RequestManager.getInstance().setProvider(provider);
 };
+web3.isConnected = function(){
```

Comment:
I think that the dapp should not care or know if its connected, that is the job of the browser.